packages/client/src/components/ClientProvider/ClientProvider.tsx

```diff
@@ -204,19 +118,70 @@ const ClientProvider = ({
               }
             },
           }),
-          dedupExchange,
```

Comment:
What happened to dedupExchange?  Is it part of the libaray now?

Reply:
Yup! 

https://github.com/urql-graphql/urql/blob/1470a9c7474c75b65303b503c0a3a51e895f1a38/packages/core/src/exchanges/dedup.ts#L3-L6